pkg/tsdb/cloudwatch/time_series_query_test.go

```diff
@@ -415,7 +417,10 @@ func Test_QueryData_response_data_frame_name_is_always_response_label(t *testing
 	t.Cleanup(func() {
 		NewCWClient = origNewCWClient
 	})
-	var api mocks.MetricsAPI
+
+	api := mocks.MetricsAPI{Metrics: []*cloudwatch.Metric{
+		{MetricName: aws.String(""), Dimensions: []*cloudwatch.Dimension{{Name: aws.String("InstanceId"), Value: aws.String("i-00645d91ed77d87ac")}}},
+	}}
```

Comment:
To use testify/mock we'll need to expect the call to ListMetricsPages here:
`	api.On("ListMetricsPages").Return(nil)`

Reply:
done